recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+make -e \
+  TOPDIR="${PREFIX}" \
+  USRDIR='' \
+  POSIXRULES='' \
+  ZFLAGS='-b slim' \
```

Comment:
I would use `-b fat` (or whatever the equivalent is) for now. `pytz` and `dateutil` will have problems with the slim versions, unfortunately.

Reply:
Ah, thanks. I assume you intend to change the Python package to `-b fat` then too?

Reply:
No, the Python package is not used by anything except `zoneinfo` at the moment, whereas `dateutil` will automatically use the system time zone information, and `pytz` has an option to use the system time zone data. Anyone who starts using the Python `tzdata` package should support slim files.